pages/api/[...mockApiRoute].ts

```diff
@@ -63,6 +64,10 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       mockApiRoute === 'sites/b97f68cd-50e0-49bb-a850-108d4a9f7b7e/pv_forecast'
     ) {
       res.status(200).json(pvForecastJson);
+    } else if (
+      mockApiRoute === 'sites/b97f68cd-50e0-49bb-a850-108d4a9f7b7e/clearsky'
```

Comment:
this route is called `clearsky_estimate` in the actual API. see
[`http://pvsite-development-api-sites.eu-west-1.elasticbeanstalk.com/sites/04b3d67f-af48-42f7-ac0f-9d07d4dd84f6/clearsky_estimate`](http://pvsite-development-api-sites.eu-west-1.elasticbeanstalk.com/sites/04b3d67f-af48-42f7-ac0f-9d07d4dd84f6/clearsky_estimate)

Reply:
bump on changing this to clearsky_estimate